.cz-config.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
     { name: 'nextjs', description: 'anything Next specific' },
     { name: 'nest', description: 'anything Nest specific' },
     { name: 'node', description: 'anything Node specific' },
+    { name: 'express', description: 'anything Express specific' },
```

Comment:
Hm `express` is supposed to be under `node` :shrug:

Reply:
Why would `nest` get it's own thing then?